### PR TITLE
Text utility

### PR DESCRIPTION
**Pull request by theVivek-kumar**
text utility done please review

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
GTM

File: text-utility/text-utility.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    padding: 10px;
+    color: rgb(70, 68, 69);
+    font-size: 3rem;
+    color: rgb(255, 60, 0);
+}
+.heading2{
+    border-bottom: 1px solid rgb(206, 206, 206);
+    padding: 10px;
+    font-size: 2rem;
+    color: rgb(172, 4, 121);
```
Comment (deekshithmd) [inline text-utility/text-utility.css:22]: using color variables will be good

File: text-utility/text-utility.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            
+                    <h3 class="heading2">h2. Small text</h3>
+            
+                    <h4 class="heading3"> This is gray text</h4>
+            
+            
+            
+                    
+                    <h3 class="typography">Center Text</h3>
+                </div>
```
Comment (deekshithmd) [inline text-utility/text-utility.html:101]: eleminate all extra spaces

Comment (theVivek-kumar) [reply text-utility/text-utility.html:101]: sure

**Pull request merged**